content/applications/websites/website/publish/on-premise_geo-ip-installation.rst

```diff
@@ -5,60 +5,55 @@ Geo IP Installation (On-Premises Database)
 Installation
 ============
 
-.. warning::
-   Please note that the installation depends on your computer's operating system and distribution.
-   We will assume that a Linux operating system is being used.
+1. Download both the GeoLite2 City and Country
+   `databases <https://dev.maxmind.com/geoip/geoip2/geolite2/>`_. You should end up with two files
+   called ``GeoLite2-City.mmdb`` and ``GeoLite2-Country.mmdb``
 
-#. Install `geoip2 <https://pypi.org/project/geoip2/>`__ Python library
-    .. code-block:: bash
-
-      pip install geoip2
-
-#. Download the `GeoLite2 City database <https://dev.maxmind.com/geoip/geoip2/geolite2/>`_. You
-   should end up with a file called ``GeoLite2-City.mmdb``
-#. Move the file to the folder ``/usr/share/GeoIP/``
+2. Move the files to the folder ``/usr/share/GeoIP/``
     .. code-block:: bash
 
         mv ~/Downloads/GeoLite2-City.mmdb /usr/share/GeoIP/
+        mv ~/Downloads/GeoLite2-Country.mmdb /usr/share/GeoIP/
```

Comment:
While you're at it, remove the extra indentation space before each of these lines that makes item 2. behave as a quote block. You'll also need to add an extra blank line after item 2.
```suggestion

   .. code-block:: bash

       mv ~/Downloads/GeoLite2-City.mmdb /usr/share/GeoIP/
       mv ~/Downloads/GeoLite2-Country.mmdb /usr/share/GeoIP/
```